lib/unexpected-knex.js

```diff
@@ -190,6 +193,34 @@ function init(expect) {
     }
   });
 
+  expect.addAssertion(
+    '<knex|knexQuery> with table <string> <assertion?>',
```

Comment:
seems like this could be added to the assertion defined on line 205. otherwise they're exactly the same :)